### ISSUE TITLE
A problem about import qso from LoTW

### ISSUE THREAD

**Issue by BH6AOV** [magicbug/Cloudlog#1303]
**This is for reporting bugs with the code, if this issue is regarding setting up on your local server please use the forum see readme for details**

**Describe the bug**
Sync QSO from LoTW and php give me error "Trying to get property 'COL_LOTW_QSLRDATE' of non-object"

**To Reproduce**
Steps to reproduce the behaviour:
Select manual sync in /lotw

**OS Infomation**
OS: `Ubuntu 18.04 LTS x64`
WebServer: `Nginx/1.14.0` (I believe this problem has nothing to do with this)
PHP: `7.3.31-2+ubuntu18.04.1+deb.sury.org+1`
Mysql: `5.7.36-0ubuntu0.18.04.1 - (Ubuntu)`

**Additional context**
This is error text:
```text
A PHP Error was encountered
Severity: Notice

Message: Trying to get property 'COL_LOTW_QSLRDATE' of non-object

Filename: models/Logbook_model.php

Line Number: 1483
```

**Comment by AndreasK79** [comment]
@BH6AOV did this work before, or is this the first time trying it?

**Comment by BH6AOV** [comment]
> @BH6AOV did this work before, or is this the first time trying it?

This is first time

**Comment by AndreasK79** [comment]
@BH6AOV thanks. That confirms what I am thinking. The problem is that COL_LOTW_QSLRDATE is not set at all.
I've pushed a fix with #1323. Could you pull the latest master and give it a try? Please report back if this fixes it.

**Comment by BH6AOV** [comment]
> @BH6AOV thanks. That confirms what I am thinking. The problem is that COL_LOTW_QSLRDATE is not set at all.
> I've pushed a fix with #1323. Could you pull the latest master and give it a try? Please report back if this fixes it.

Ok, i will try it and report in here.

**Comment by AndreasK79** [comment]
@BH6AOV did it work?
